plugin/src/App/getTextBoundsAsync.lua

```diff
@@ -0,0 +1,37 @@
+local TextService = game:GetService("TextService")
+
+local Rojo = script:FindFirstAncestor("Rojo")
+local Packages = Rojo.Packages
+
+local Log = require(Packages.Log)
+
+local params = Instance.new("GetTextBoundsParams")
+
+local function getTextBoundsAsync(
+	text: string,
+	font: Font,
+	textSize: number,
+	width: number,
+	richText: boolean?
+): Vector2
+	if type(text) ~= "string" or #text > 200_000 then
```

Comment:
`#text > 200_000` needs to be either `#text >= 200_000` or `#text > 199_999` here, since the limit is 199,999. Sorry, I could have been more clear in my initial comment here!

Also, it might not be a great idea to send a massive string through `Log.warn` - in cases where the text is too long, we might just want to say "text too long" without printing the actual string